app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -92,14 +96,12 @@ def check_enrollment(enrollment)
   rescue Faraday::BadRequestError => err
     # 400 status code. This is used for some status updates and some common client errors
     handle_bad_request_error(err, enrollment)
-  rescue Faraday::ClientError, Faraday::ServerError => err
-    # 4xx or 5xx status code. These are unexpected but will have some sort of
-    # response body that we can try to log data from
+  rescue Faraday::ClientError, Faraday::ServerError, Faraday::Error => err
+    # 4xx or 5xx status code.
+    # These are unexpected but will have some sort of response body that we can try to log data from
+    # Faraday Timeouts, failed connections, parsing errors, and other HTTP errors.
+    # These generally won't have a response body
     handle_client_or_server_error(err, enrollment)
```

Comment:
The comment on the new code doesn't cohesively combine the previous comments giving context on these errors.

Reply:
WIll fix.